release-tools/stochss_ami_config.json

```diff
@@ -0,0 +1,113 @@
+{
+    "base_ami_id": "ami-00ab0168",
+    "aws_region": "us-east-1",
+    "instance_type": "c3.large",
+    "instance_user": "ubuntu",
+    "git_repo": {
+        "url": "https://github.com/StochSS/stochss.git",
+        "branch": "cost-analysis"
```

Comment:
Yes, I know ... but for testing it, I'm keeping it like that. It should be updated when we have the final ami's ready.